tests/hmm/hmumu_utils.py

```diff
@@ -1227,7 +1227,11 @@ def get_selected_muons(
     """
 
     if fsrphotons:
+        out_muons_pt = correct_muon_with_fsr(muons.offsets, fsrphotons.offsets,
+                              muons.pt, muons.eta, muons.phi, muons.mass, muons.pfRelIso04_all, muons.fsrPhotonIdx,
+                              fsrphotons.pt, fsrphotons.eta, fsrphotons.phi)
 
+    if False:
         out_muons_fsrPhotonIdx = NUMPY_LIB.array(muons.fsrPhotonIdx)
         fix_muon_fsrphoton_index(fsrphotons.offsets, muons.offsets, fsrphotons.dROverEt2, fsrphotons.muonIdx, muons.fsrPhotonIdx, out_muons_fsrPhotonIdx)
```

Comment:
I think this still needs to be enabled - @irenedutta23 found that there are muons with multiple FSR photons matched, in which case the index in NanoAOD needs to be updated.